src/components/Popup/Popup.tsx

```diff
@@ -1,5 +1,4 @@
 import React, { useState } from 'react';
-import ReCAPTCHA from 'react-google-recaptcha';
```

Comment:
Is ReCAPTCHA only removed for testing purposes or will it be removed for the future of the project? Since this change will remain in the pull request to the main repository when that happens too.

Reply:
This will have to be removed for presentation but be reintroduced later